lang/frontend/ide/src/main/java/tools/vitruv/neojoin/ide/Main.java

```diff
@@ -31,15 +31,15 @@ public class Main implements Runnable {
 
 	private static final String NoValue = "__NO_VALUE__"; // option is specified but without a value
 
-	@Option(names = {"-m", "--meta-model-path"}, paramLabel = "MODEL PATH", required = true, description = "Path specification to find referenced meta-models")
+	@Option(names = {"-m", "--meta-model-path"}, paramLabel = "MODEL-PATH", required = true, description = "Path specification to find referenced meta-models")
 	String metaModelPath;
 
 	@ArgGroup(exclusive = false)
 	Logging logging;
 
 	static class Logging {
 
-		@Option(names = {"--log"}, paramLabel = "LOG FILE", required = true, arity = "0..1", fallbackValue = NoValue, description = "Enable logging")
+		@Option(names = {"--log"}, paramLabel = "LOG-FILE", required = true, arity = "0..1", fallbackValue = NoValue, description = "Enable logging")
```

Comment:
Would it make sense to use Java `Optional<T>` (see https://picocli.info/#_optionalt) instead of a hard coded `NoValue`?